### PR TITLE
news216: add French translation 

### PR DESCRIPTION
**Pull request by Copinmalin**
It's my first time so please share me if something wrong, and how can I improve myself

### CODE REVIEW

File: _posts/fr/newsletters/2022-09-07-newsletter.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
+---
+title: 'Bitcoin Optech Newsletter #216'
+permalink: /fr/newsletters/2022/09/07/
+name: 2022-09-07-newsletter
+slug: 2022-09-07-newsletter
```
Comment (tschweg) [inline _posts/fr/newsletters/2022-09-07-newsletter.md:5]: I think you need to add a '-fr' suffix to the _slug_ and _name_ properties or the make file will throw errors.

Comment (Copinmalin) [reply _posts/fr/newsletters/2022-09-07-newsletter.md]: Done, 
but `make production` is still in error :
bundle exec _contrib/schema-validator.rb _data/schemas/compatibility.yaml _data/compatibility/blockchaininfo.yaml
/usr/bin/env: ‘ruby\r’: No such file or directory
make: *** [Makefile:120: _site/en/compatibility/blockchaininfo/index.html] Error 127

Comment (Copinmalin) [reply _posts/fr/newsletters/2022-09-07-newsletter.md]: `bundle exec jekyll serve` is working fine

Comment (tschweg) [reply _posts/fr/newsletters/2022-09-07-newsletter.md]: Looks like the issue is trailing spaces. Make sure to remove all of them.
I was able to build your version after fixing the spaces and running _make clean_ and then _make_

Comment (Copinmalin) [reply _posts/fr/newsletters/2022-09-07-newsletter.md]: I have an error : 
cyrille@PC-Commun:/mnt/c/Users/cyril/OneDrive/Documents/GitHub/bitcoinops.github.io$ make
bundle exec _contrib/schema-validator.rb _data/schemas/compatibility.yaml _data/compatibility/blockchaininfo.yaml
/usr/bin/env: ‘ruby\r’: No such file or directory
make: *** [Makefile:120: _site/en/compatibility/blockchaininfo/index.html] Error 127

Comment (Copinmalin) [reply _posts/fr/newsletters/2022-09-07-newsletter.md]: But it's work well with this command : ` bundle exec jekyll serve`

File: Gemfile.lock [hunk 2]
```diff
@@ -108,3 +108,6 @@ DEPENDENCIES
 
 RUBY VERSION
    ruby 2.6.4p104
+
+BUNDLED WITH
```
Comment (bitschmidty) [inline Gemfile.lock:112]: Im not sure if this is required or not

File: fr/topics.md [hunk 3]
```diff
@@ -0,0 +1,64 @@
+---
+title: Topics-fr
```
Comment (bitschmidty) [inline fr/topics.md:2]: Do you plan to add french topics @Copinmalin ?

I dont believe we have tested translations of topics yet. So if you plan to do that in the near term, we can work on making sure that works. And if not in the short term, maybe we wait on topic changes? 

(Note: other translations do not have <language>/topics.md yet

Comment (Copinmalin) [reply fr/topics.md:2]: Hello Mike,

no I don't plan to add french topic for now, 
if necessary I can remove this folder

File: _posts/fr/newsletters/2022-09-07-newsletter.md [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+
+- [Bitcoin Core #25355][] Ajout du support pour la transition, un adresse
+  temporaire I2P seulement quand la sortie [I2P connections][topic anonymity
+  networks] est autorisée. Dans I2P, celui qui reçoit apprend l'adresse I2P de
+  la connexion de l'initiateur. Les noeuds I2P non listés utiliseront maintenant
+  par défaut les adresses I2P temporaires lorsqu'ils établissent des connexions
+  sortantes.
+
+- [BDK #689][] ajout d'une méthode `allow_dust` autorisant un portefeuille à
+  créer une transaction qui viole la limite de poussière [dust limit][topic
```
Comment (jonatack) [inline _posts/fr/newsletters/2022-09-07-newsletter.md:65]: suggest replacing "viole" with another verb; maybe "ne respecte pas"

**Review by jonatack — COMMENTED** [review]
Nice to see a French translation! A few thoughts after a quick read-through.

File: _posts/fr/newsletters/2022-09-07-newsletter.md [hunk 5]
```diff
@@ -0,0 +1,82 @@
...
+title: 'Bitcoin Optech Newsletter #216'
+permalink: /fr/newsletters/2022/09/07/
+name: 2022-09-07-newsletter-fr
+slug: 2022-09-07-newsletter-fr
+type: newsletter
+layout: newsletter
+lang: fr
+---
+Cette semaine au sommaire de cette newsletter, plusieurs changements notable au
+populaire logiciel de l'infrastructure Bitcoin.
```
[the remark this answers is not in the document]
Comment (Copinmalin) [reply _posts/fr/newsletters/2022-09-07-newsletter.md:11]: "principal logiciel" could be acceptable ?

[the remark this answers is not in the document]
Comment (jonatack) [reply _posts/fr/newsletters/2022-09-07-newsletter.md:11]: Optech covers several projects.

File: _posts/fr/newsletters/2022-09-07-newsletter.md [hunk 6]
```diff
@@ -0,0 +1,82 @@
...
+  accumulé rencontre un seuil prédéterminé, puis une seconde passe pour les
+  mémoriser. Afin de prévenir qu'un attaquant envoie la chaîne principale lors
+  de la pré-synchronisation et ensuite une différente, une chaîne malicieuse
+  lors du re-téléchargement, le noeud stocke les engagements envers la chaîne
+  d'en-têtes pendant la pré-synchronisation.
+
+- [Bitcoin Core #25355][] Ajout du support pour la transition, un adresse
+  temporaire I2P seulement quand la sortie [I2P connections][topic anonymity
+  networks] est autorisée. Dans I2P, celui qui reçoit apprend l'adresse I2P de
+  la connexion de l'initiateur. Les noeuds I2P non listés utiliseront maintenant
```
[the remark this answers is not in the document]
Comment (Copinmalin) [reply _posts/fr/newsletters/2022-09-07-newsletter.md]: I changed by "n'accepte pas les connexions entrantes" right ?

File: fr/topics.md (hunk 3 above)
Comment (bitschmidty) [reply fr/topics.md]: Ok, can you remove for now? Thanks!

File: Gemfile.lock (hunk 2 above)
Comment (bitschmidty) [reply Gemfile.lock:112]: @Copinmalin would you mind removing this? I prefer not to introduce any potential issues as long as you can build locally without it, etc.

Comment (Copinmalin) [reply Gemfile.lock]: Done

**Comment by bitschmidty** [comment]
@Copinmalin Sorry, I meant just delete the change to the gemfile.lock file, not the entire file :)

File: fr/blog.md [hunk 7]
```diff
@@ -0,0 +1,9 @@
+---
+layout: blog
+lang: fr
+title: Blog-fr
+name: blog-fr
+permalink: /fr/blog/
+share: false
+version: 1
+---
```
Comment (bitschmidty) [inline fr/blog.md:9]: For consistency to other translations, can you add a French translation text of this placeholder text for this page?

> Would you like to help translate our publications? See the [CONTRIBUTING documentation](https://github.com/bitcoinops/bitcoinops.github.io/blob/master/CONTRIBUTING.md#translations) and the [French translation issues and PRs](https://github.com/bitcoinops/bitcoinops.github.io/pulls?&q=label%3Alocalization-french) in our github repo.

**Review by bitschmidty — CHANGES_REQUESTED** [review]

File: fr/blog.md (hunk 7 above)
Comment (Copinmalin) [reply fr/blog.md:9]: OK, done

Comment (bitschmidty) [reply fr/blog.md:9]: Pushed a commit to put in the links on this page

Comment (Copinmalin) [reply fr/blog.md:9]: Hello,

Does all is OK now ?

**Review by bitschmidty — APPROVED** [review]
Pushed some small link fixups.

Everything looking good now. Thank you so much for this first French translation, @Copinmalin ! 🚀 👏 🎉

File: _posts/fr/newsletters/2022-09-07-newsletter.md [hunk 8]
```diff
@@ -0,0 +1,82 @@
...
+  Avec ces changements, les noeuds téléchargent deux fois les en-têtes durant la
+  synchronisation de l'en-tête initiale: une première fois pour vérifier
+  l'en-tête de la preuve de travail (sans la stocker) jusqu'à ce que le travail
+  accumulé rencontre un seuil prédéterminé, puis une seconde passe pour les
+  mémoriser. Afin de prévenir qu'un attaquant envoie la chaîne principale lors
+  de la pré-synchronisation et ensuite une différente, une chaîne malicieuse
+  lors du re-téléchargement, le noeud stocke les engagements envers la chaîne
+  d'en-têtes pendant la pré-synchronisation.
+
+- [Bitcoin Core #25355][] Ajout du support pour la transition, un adresse
```
[the remark this answers is not in the document]
Comment (jonatack) [reply _posts/fr/newsletters/2022-09-07-newsletter.md]: Two issues with "seulement quand la sortie I2P connections est autorisée"

- grammar: "sortie I2P connections" should be "sortie de connexions I2P"

- meaning: this means "only when outbound connections are authorized" ... the meaning needs to be "when the node is configured to only make outbound connections (i.e. no inbound ones)"

File: _posts/fr/newsletters/2022-09-07-newsletter.md [hunk 9]
```diff
@@ -0,0 +1,82 @@
...
+Interface (HWI)][hwi repo], [Rust Bitcoin][rust bitcoin repo], [BTCPay
+Server][btcpay server repo], [BDK][bdk repo], [Bitcoin Improvement
+Proposals (BIPs)][bips repo], and [Lightning BOLTs][bolts repo].*
+
+- [Bitcoin Core #25717][] Ajout d'une étape de pré-synchronisation des entêtes
+  (Headers Presync) durant le téléchargement initial des blocs (Initial Block
+  Download - IBD) afin de prévenir d'une attaque par déni de service (DoS) ainsi
+  qu'une étape vers la suppression des points de contrôle. Les noeuds utilisent
+  la pré-synchronisation pour vérifier que la chaîne d'en-têtes d'un pair a
+  suffisamment travaillé avant de la stocker de manière permanente.
```
Comment (jonatack) [inline _posts/fr/newsletters/2022-09-07-newsletter.md:31]:
```diff
 - [Bitcoin Core #25717][] Ajout d'une étape de pré-synchronisation des entêtes
   (Headers Presync) durant le téléchargement initial des blocs (Initial Block
   Download - IBD) afin de prévenir d'une attaque par déni de service (DoS) ainsi
-  qu'une étape vers la suppression des points de contrôle. Les noeuds utilisent
+  qu'une étape vers la suppression des points de contrôle de consensus (chainparams checkpoints). Les noeuds utilisent
   la pré-synchronisation pour vérifier que la chaîne d'en-têtes d'un pair a
-  suffisamment travaillé avant de la stocker de manière permanente.
+  suffisamment de travail cumulé (proof of work) avant de la stocker de manière permanente.
```

File: _posts/fr/newsletters/2022-09-07-newsletter.md [hunk 10]
```diff
@@ -0,0 +1,82 @@
...
+
+  Au cours de l’IBD, un pair malicieux pourrait essayer de bloquer le process de
+  synchronisation, injecter des blocs qui ne sont pas de la chaîne la plus
+  longue, ou simplement épuiser les ressources du noeud. Ainsi, bien que la
+  vitesse de synchronisation et l'usage de la bande passante soient des
+  préoccupations importantes pendant l'IBD, un objectif principal de conception
+  est d'éviter l'attaque par déni de service (DoS). Depuis la version v0.10.0,
+  le noeud Bitcoin Core synchronise d'abord la première en-tête de block avant
+  de télécharger les données du bloc et rejette les en-têtes qui ne respectent
+  pas un ensemble de points de contrôle.
```
Comment (jonatack) [inline _posts/fr/newsletters/2022-09-07-newsletter.md:41]: It's great that you are doing this @Copinmalin. That said, I would encourage you to check carefully for inaccuracies and errors a few hours or a day later after getting some distance. It's not clear to me whether these are issues with translation  or misunderstanding of some of the nuance in the source content. A few examples in one paragraph follow. Thanks for the excellent initiative!

```diff
-  Au cours de l’IBD, un pair malicieux pourrait essayer de bloquer le process de
+  Au cours de l’IBD, un pair malicieux pourrait essayer de ralentir le process de
   synchronisation, injecter des blocs qui ne sont pas de la chaîne la plus
   longue, ou simplement épuiser les ressources du noeud. Ainsi, bien que la
   vitesse de synchronisation et l'usage de la bande passante soient des
-  préoccupations importantes pendant l'IBD, un objectif principal de conception
-  est d'éviter l'attaque par déni de service (DoS). Depuis la version v0.10.0,
-  le noeud Bitcoin Core synchronise d'abord la première en-tête de block avant
-  de télécharger les données du bloc et rejette les en-têtes qui ne respectent
-  pas un ensemble de points de contrôle.
+  préoccupations importantes pendant l'IBD, un objectif principal reste
+  l'évitement des attaques par déni de service (DoS). Depuis la version v0.10.0,
+  Bitcoin Core synchronise d'abord les en-têtes des blocks avant
+  de télécharger les données des blocks et rejette les en-têtes qui ne respectent
+  devient des checkpoints de consensus.
```

**Pull request merged**